dataprofiler/profilers/profiler_options.py

```diff
@@ -1570,6 +1572,8 @@ def __init__(self, presets: str = None) -> None:
                 self._data_types_presets()
             elif self.presets == "numeric_stats_disabled":
                 self._numeric_stats_disabled_presets()
+            elif self.presets == "memory_optimization":
+                self._memory_optimization_presets()
```

Comment:
let's add some validation in `_validate_helper` for the universe of `presets` options that are valid. It should fail if not valid preset option